### PR TITLE
[remove:1] Switch to promises

### PR DESCRIPTION
**Pull request by d2lam**
Want to switch to promises first before I add `_remove` function to `datastore-dynamo`. 

Needed for https://github.com/screwdriver-cd/models/issues/105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,24 +17,25 @@ npm install screwdriver-datastore-base
...
+    _get(config) {
 
         // do something to fetch data...
 
         if (config.params.id > 0) {
-            return callback(null, { id: config.params.id });
+            return Promise.resolve({ id: config.params.id });
         }
 
         return process.nextTick(() => {
```
Comment (stjohnjohnson) [inline README.md]: `process.nextTick` will not trickle the return down.  Just remove it.

**Review by stjohnjohnson — CHANGES_REQUESTED** [review]
Just an update of the README file

File: README.md (hunk 1 above)
Comment (d2lam) [reply README.md]: @stjohnjohnson  Done

**Review by stjohnjohnson — APPROVED** [review]
Looks good!

**Pull request merged**